API_GUIDE.md

```diff
@@ -152,6 +160,25 @@ See the
 [full multiprocessing example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist.py)
 for more on training a network on multiple XLA devices with multi-processing.
 
+### Running on Multiple XLA Hosts
+Multi-host setup for different accelerators can be very different. This doc will talk about the device independent bits of multi-host training and will use the TPU + PJRT runtime(currently available on 1.13 and 2.x releases) as an example.
+
+Let's assume you have the above mnist example from above section in a `train_mnist_xla.py`. If it is a single host multi device training, you would run it like
+
+```
+PJRT_DEVICE=TPU python3 train_mnist_xla.py
+```
+
+Now in order to run the same models on a TPU v4-16 (which has 2 host, each with 4 TPU devices), you will need to
+  - Make sure each host can access the training script and training data. This is usually done by using the `gcloud scp` command or `gcloud ssh` command to copy the training scripts to all hosts.
```

Comment:
First mention of gcloud. Maybe add pointer about how to reserve TPUs here or at the beginning.

Reply:
same, I share the link for TPU pod doc link at the end of this section, do you think it is better to move that up here?

Reply:
Yea, I think it's better to move the link about. Should we mention shared disk here?

Reply:
I will move the doc to the top, I think share disk is a bit too detial, we should probably mention that in the user guide on cloud dev site.